recipe/meta.yaml

```diff
@@ -50,6 +90,41 @@ test:
     - prefect.tasks.shell
     - prefect.tasks.control_flow
     - prefect.tasks.github
+  requirements:
+    run:
+      {{ list_requirements(sdata['extras_require']['test']) | indent(4) }}
+
+output:
+  # The main build
+  - name: prefect
+  # The development build
+  - name: prefect-dev
```

Comment:
You can drop the dev build, that extra is more for local development of prefect and CI than anything a user should need to do.